bin/setup

```diff
@@ -15,7 +15,8 @@ FileUtils.chdir APP_ROOT do
 
   puts "== Installing dependencies =="
   system! "gem install bundler --conservative"
-  system! "bundle config --local set cache_all true"
+  system! "bundle config --local cache_all true"
+  system! "bundle config --local cache_all_platforms true"
```

Comment:
For anyone curious, we decided to add this since some gems (e.g. `libv8`) install platform specific versions like `libv8-7.3.492.27.1-x86_64-linux.gem`.